cmd/embed/main.go

```diff
@@ -51,6 +52,7 @@ func main() {
 	flag.StringVar(&pkg, "package", "", "the package to use, defaults to the dir name")
 	flag.StringVar(&output, "out", "embed.go", "the file to generate")
 	flag.StringVar(&root, "root", "", "The root path for embedding")
+	flag.BoolVar(&web, "web", false, "If true, replace back slash separators in filepaths with forward slashes")
```

Comment:
My thinking was that embed should keep the path separator of the host system in most cases since they represent the actual offsets of the files they come from on those systems. It's just embedded web pages that want to be able to look up by those paths from a web url. The other option I considered was having the static file webserver "fixup" the incoming url's path separators to whatever the host used (like the fix for the root index), which would probably be the simplest and easiest solution of all. Let me know which option sounds best.